getting-started/git-boot-camp.rst

```diff
@@ -283,6 +283,14 @@ Creating a Pull Request
 
 6. Press the ``Create pull request`` button.
 
+You should include the issue number in the title of the PR (``gh-NNNNN: ...``).
```

Comment:
```suggestion
You should include the issue number in the title of the PR,
in the format ``gh-NNNNN: <PR Title>``.
```

Be more specific regarding the format, since at least on the most of the rest of GitHub itself, the conventional way to refer to GitHub issues numbers is using `#` rather than `gh-`.